pkg/ipamd/ipamd.go

```diff
@@ -850,8 +953,29 @@ func getWarmENITarget() int {
 	return defaultWarmENITarget
 }
 
-func logPoolStats(total, used, maxAddrsPerENI int) {
-	log.Debugf("IP pool stats: total = %d, used = %d, c.maxIPsPerENI = %d", total, used, maxAddrsPerENI)
+func getWarmPrefixTarget() int {
+	inputStr, found := os.LookupEnv(envWarmPrefixTarget)
+
+	if !found {
+		return noWarmPrefixTarget
+	}
+
+	if input, err := strconv.Atoi(inputStr); err == nil {
+		if input < 0 {
+			return noWarmPrefixTarget
+		}
+		log.Debugf("Using WARM_PREFIX_TARGET %v", input)
+		return input
+	}
+	return noWarmPrefixTarget
+}
+
+func logPoolStats(total int, used int, maxAddrsPerENI int, Ipv4PrefixDelegation bool) {
+	if !Ipv4PrefixDelegation {
+		log.Debugf("IP pool stats: total = %d, used = %d, c.maxIPsPerENI = %d", total, used, maxAddrsPerENI)
+	} else if Ipv4PrefixDelegation {
```

Comment:
```suggestion
	} else {
```

.. also - I think these two branches are identical??

Reply:
Ahh my bad, wanted to add prefix related debug.